projects/core/src/config/config-initializer/config-initializer.module.ts

```diff
@@ -30,8 +30,11 @@ export function configInitializerFactory(
 
 export function locationInitializedFactory(
   configInitializer: ConfigInitializerService
-): Promise<Config> {
-  return lastValueFrom(configInitializer.getStable());
+): () => Promise<Config> {
```

Comment:
Too bad, this function is in public API, so we shouldn't change its signature from `Promise` to `()=>Promise`